rmq/rmqreceiver/rabbitmq_receiver.py

```diff
@@ -62,6 +63,7 @@ def __init__(self, consumer_callback, amqp_url, exchange, **kwargs):
         self._url = amqp_url
         self.exchange = exchange
         self.parse_input_args(kwargs)
+        self.prev_backoff = 1
```

Comment:
shouldn't be global... will create problem in cases where application is not frequently deployed.